diff -Naur linux-5.14.0-70.30.1.el9_0/tools/testing/selftests/memfd/memfd_test.c linux-5.14.0-70.30.1.el9_0.alma/tools/testing/selftests/memfd/memfd_test.c
--- linux-5.14.0-70.30.1.el9_0/tools/testing/selftests/memfd/memfd_test.c	2022-10-14 13:02:21.000000000 +0000
+++ linux-5.14.0-70.30.1.el9_0.alma/tools/testing/selftests/memfd/memfd_test.c	2022-11-03 12:45:19.583010255 +0000
@@ -6,7 +6,7 @@
 #include <inttypes.h>
 #include <limits.h>
 #include <linux/falloc.h>
-#include <fcntl.h>
+#include <linux/fcntl.h>
 #include <linux/memfd.h>
 #include <sched.h>
 #include <stdio.h>
diff -Naur linux-5.14.0-70.30.1.el9_0/tools/testing/selftests/vm/userfaultfd.c linux-5.14.0-70.30.1.el9_0.alma/tools/testing/selftests/vm/userfaultfd.c
--- linux-5.14.0-70.30.1.el9_0/tools/testing/selftests/vm/userfaultfd.c	2022-10-14 13:02:21.000000000 +0000
+++ linux-5.14.0-70.30.1.el9_0.alma/tools/testing/selftests/vm/userfaultfd.c	2022-11-03 12:46:20.128372395 +0000
@@ -46,6 +46,7 @@
 #include <signal.h>
 #include <poll.h>
 #include <string.h>
+#include <linux/mman.h>
 #include <sys/mman.h>
 #include <sys/syscall.h>
 #include <sys/ioctl.h>
diff -Naur linux-5.14.0-70.30.1.el9_0/tools/testing/selftests/bpf/Makefile linux-5.14.0-70.30.1.el9_0.alma/tools/testing/selftests/bpf/Makefile
--- linux-5.14.0-70.30.1.el9_0/tools/testing/selftests/bpf/Makefile	2022-10-14 13:02:21.000000000 +0000
+++ linux-5.14.0-70.30.1.el9_0.alma/tools/testing/selftests/bpf/Makefile	2022-11-07 20:12:15.536377539 +0000
@@ -279,6 +279,7 @@
 CLANG_SYS_INCLUDES = $(call get_sys_includes,$(CLANG))
 BPF_CFLAGS = -g -D__TARGET_ARCH_$(SRCARCH) $(MENDIAN) 			\
 	     -I$(INCLUDE_DIR) -I$(CURDIR) -I$(APIDIR)			\
+	     -I$(abspath $(TOOLSDIR)/../usr/include)			\
 	     -I$(abspath $(OUTPUT)/../usr/include)
 
 CLANG_CFLAGS = $(CLANG_SYS_INCLUDES) \
